applications/luci-app-wireguard/luasrc/view/wireguard.htm

```diff
@@ -84,11 +84,11 @@
 		var seconds = (now.getTime() / 1000) - timestamp;
 		var ago = "";
 		if (seconds < 60) {
-			ago = parseInt(seconds) + '<%:s ago%>';
+			ago = parseInt(seconds) + ' <%:s ago%>';
 		} else if (seconds < 3600) {
-			ago = parseInt(seconds / 60) + '<%:m ago%>';
+			ago = parseInt(seconds / 60) + ' <%:min ago%>';
 		} else if (seconds < 86401) {
-			ago = parseInt(seconds / 3600) + '<%:h ago%>';
+			ago = parseInt(seconds / 3600) + ' <%:h ago%>';
```

Comment:
Is the extra space intentional?

Reply:
Yes. All units should have a space between the quantity and the unit symbol, both grammatically and as per the SI.

Reply:
While obviously it's not that big of a deal on a greater scheme of things, if this results in `(5 s ago)` rather than `(5s ago)` I'm not sure if it's an improvement.

Reply:
I understand it seems strange at first, but "5s ago" means "fives ago", whereas "5 s ago" means "5 seconds ago". I think it would make more sense if you look at it with minutes: "5 min ago" is the proper way while "5min ago" looks incorrect.